docs/getting-started/documentation/pull-data-transfer.md

```diff
@@ -72,17 +72,19 @@ To start a pull-http-transfer using the management-API of the EDC one can send t
 `POST` to `{{connector-base-url}}/control/data/transferprocess`
 ```json
 {
-  "protocol": "ids-multipart",
+  "@context": {
+    "edc": "https://w3id.org/edc/v0.0.1/ns/"
+  },
+  "@type": "TransferRequest",
+  "protocol": "dataspace-protocol-http",
   "assetId": "urn:artifact:http-pull",
   "contractId": "{{contract-id}}",
   "dataDestination": {
-    "properties": {
-        "type": "HttpProxy",
-        "assetId": "urn:artifact:http-pull"
-    }
+    "@type": "DataAddress",
+    "type": "HttpProxy"
```

Comment:
I am quite sure that there is URL prefixes missing, compare to `de.sovity.edc.extension.e2e.connector.DataTransferTestUtil#buildDataAddressProperties`